### DISCUSSION TITLE
Use Coral TPU?

### DISCUSSION THREAD

**Discussion by brunopiras** [skrashevich/double-take#139] [Q&A] [answered]
Hi all,

do you think can I use TPU Coral on facial recognition?

I use that in frigate (not for face rec, off course).

Now I've CodeProject.AI and DeepStack working in double-take.

Thanks

**Comment by skrashevich** [comment, marked as answer]
Actually, I don't know any way to do it

**Reply by brunopiras** [reply]
Thanks, 

I think you've done an awesome job since now!!

I'm just at the beginning experience, just two days!! 👌🏽

**Reply by krzakx** [reply]
Me too looking forward to utilize Google Coral Tensor Flow technology and models. Hope some could put more brightness on it.

**Reply by ncodee** [reply]
Just checking, did you find a way since?
